packages/share/example/lib/main.dart

```diff
@@ -83,4 +124,51 @@ class DemoAppState extends State<DemoApp> {
           )),
     );
   }
+
+  Widget _buildImagePreviews() {
```

Comment:
Why not extract these functions (buildImagePreviews and buildImagePreview) to stateless widgets in a separate image_preview.dart file?

Reply:
Good point, makes it more readable. Done so now :)